packages/app/src/cli/services/dev/update-extension.ts

```diff
@@ -50,12 +50,20 @@ export async function updateExtensionDraft({
     config = (await extension.deployConfig({apiKey, developerPlatformClient, appConfiguration})) || {}
   }
 
+  let draftableConfig: {[key: string]: unknown} = {
+    ...config,
+    serialized_script: encodedFile,
+  }
+  if (extension.isFunctionExtension) {
+    const compiledFiles = await readFile(extension.outputPath, {encoding: 'base64'})
+    draftableConfig = {
+      ...draftableConfig,
+      uploaded_files: {'index.wasm': compiledFiles},
+    }
+  }
   const extensionInput: ExtensionUpdateDraftMutationVariables = {
     apiKey,
-    config: JSON.stringify({
-      ...config,
-      serialized_script: encodedFile,
-    }),
+    config: JSON.stringify(draftableConfig),
```

Comment:
Does this mean we can remove the custom wasm update then? or are we waiting until this is released and used for a bit?

Reply:
Yes, the latter!